src/development/platform-integration/platform-views.md

```diff
@@ -317,13 +317,9 @@ import io.flutter.plugin.platform.PlatformViewFactory;
 import java.util.Map;
 
 class NativeViewFactory extends PlatformViewFactory {
-  @NonNull private final BinaryMessenger messenger;
-  @NonNull private final View containerView;
```

Comment:
I was trying to find why these were added in the first place, but couldn't see anything. 

This  matches the Kotlin example, and it's correct.

Just a comment, please remove the imports as well.